shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -55,6 +55,14 @@ typedef struct {
   const char** switches;
   // The number of elements in |switches|.
   size_t switches_count;
+  // The optional entry point in the Dart project, if the entry point is null,
+  // defaults to main().
+  const char* entry_point;
+  // Number of elements in the array passed in as dart_entrypoint_argv.
+  int dart_entrypoint_argc;
+  // Array of Dart entrypoint arguments. This is deep copied during the call
+  // to FlutterDesktopEngineCreate.
+  const char** dart_entrypoint_argv;
```

Comment:
Do you need these? These were originally added to the Windows embedder to allow passing CLI arguments. I think what you only need is to set `custom_dart_entrypoint`.

Reply:
Sorry, It's my mistake, I forgot to smbmit some code.
we need these two arguments, Dart_entrypoint_argv passed as a List<String> to Dart's entrypoint function.

Reply:
Okay. If it was intentional and you're going to use the arguments later, I'm fine.

Reply:
Yes, I have checked the engine code, and find it will be used when run entrypoint.
https://github.com/flutter-tizen/engine/blob/flutter-2.2.1-tizen/runtime/dart_isolate.cc#L188-L194
```cpp
 if (!isolate->RunFromLibrary(dart_entrypoint_library,       //
                               dart_entrypoint,               //
                               settings.dart_entrypoint_args  //
                               )) {
    FML_LOG(ERROR) << "Could not run the run main Dart entrypoint.";
    return {};
  }
```